addons/survey/models/survey_user_input.py

```diff
@@ -685,6 +685,8 @@ def _check_answer_type_skipped(self):
     @api.model_create_multi
     def create(self, vals_list):
         for vals in vals_list:
+            if vals.get('value_char_box'):
+                vals['value_char_box'] = vals['value_char_box'].strip()
```

Comment:
I wonder if this should also be added to the UI for the user to have this feedback if we do this later anyway?

Reply:
Yes, it is what I meant, and it is doable :) We can chat about it.